src/components/metadata/drag-preview.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const React = require('react')
+const cx = require('classnames')
+const { arrayOf, shape, string } = require('prop-types')
+
+
+class FieldDragPreview extends React.PureComponent {
+  get classes() {
+    return ['metadata-field', 'drag-preview', 'copy']
+  }
+
+  render() {
+    const item = this.props.items[0]
+    return (
+      <div className={cx(this.classes)}>
+        <div>
+          {item.text}
+          {item.isMixed && <span>+</span>}
+        </div>
+        {item.itemsSelected.length > 1 &&
+          <div className="badge">{item.itemsSelected.length}</div>
+        }
+      </div>
+    )
+
+  }
+
+  static propTypes = {
+    items: arrayOf(shape({
```

Comment:
See above. Let's call the prop `field` instead of `items` and do not expect an array.